app/src/main/java/de/dhbw/corona_world_app/ui/tools/StatisticCallRecyclerViewFragment.java

```diff
@@ -67,7 +64,8 @@ public void deleteItems(Set<Integer> ItemIds) {
                     statisticCallViewModel.deleteItems(ItemIds, getDataType()).get();
                 } catch (ExecutionException | InterruptedException e) {
                     Log.e(this.getClass().getName(), "items could not be deleted", e);
-                    //TODO handle
+                    //this should not happen
+                    ErrorDialog.createBasicErrorDialog(getContext(),"An critical Error has occurred","It seems that something unexpected happened, please restart the App and if that does not help reinstall the App",null);
```

Comment:
Same problem as above. "Please reinstall" seems a tad extreme... I don't know if a user who gets this message will ever reinstall the app.